### PR TITLE
[card_media_player] added standby logic

### PR DESCRIPTION
**Pull request by mdb17**
Added logic to treat the standby state the same as the off state.

### CODE REVIEW

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@
 card_media_player:
   template:
     - "icon_more_info_new"
+    - "ulm_actions_card"
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:6]: branch not up to date

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 2]
```diff
@@ -94,20 +116,16 @@ card_media_player:
...
+              variables:
+                ulm_icon_tap_action: "[[[ return variables.ulm_icon_tap_action; ]]]"
+                ulm_icon_tap_navigate_path: "[[[ return variables.ulm_icon_tap_navigate_path; ]]]"
+                ulm_icon_hold_action: "[[[ return variables.ulm_icon_hold_action; ]]]"
+                ulm_icon_hold_navigate_path: "[[[ return variables.ulm_icon_hold_navigate_path; ]]]"
+                ulm_icon_double_tap_action: "[[[ return variables.ulm_icon_double_tap_action; ]]]"
+                ulm_icon_double_tap_navigate_path: "[[[ return variables.ulm_icon_double_tap_navigate_path; ]]]"
+                ulm_input_select_option: "[[[ return variables.ulm_input_select_option; ]]]"
+                ulm_input_select: "[[[ return variables.ulm_input_select; ]]]"
+                ulm_card_media_player_enable_popup: "[[[ return variables.ulm_card_media_player_enable_popup; ]]]"
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:128]: branch not up to date

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 3]
```diff
@@ -149,20 +167,16 @@ card_media_player:
...
+              variables:
+                ulm_name_tap_action: "[[[ return variables.ulm_name_tap_action; ]]]"
+                ulm_name_tap_navigate_path: "[[[ return variables.ulm_name_tap_navigate_path; ]]]"
+                ulm_name_hold_action: "[[[ return variables.ulm_name_hold_action; ]]]"
+                ulm_name_hold_navigate_path: "[[[ return variables.ulm_name_hold_navigate_path; ]]]"
+                ulm_name_double_tap_action: "[[[ return variables.ulm_name_double_tap_action; ]]]"
+                ulm_name_double_tap_navigate_path: "[[[ return variables.ulm_name_double_tap_navigate_path; ]]]"
+                ulm_input_select_option: "[[[ return variables.ulm_input_select_option; ]]]"
+                ulm_input_select: "[[[ return variables.ulm_input_select; ]]]"
+                ulm_card_media_player_enable_popup: "[[[ return variables.ulm_card_media_player_enable_popup; ]]]"
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:179]: branch not up to date

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 4]
```diff
@@ -48,7 +49,14 @@ card_media_player:
...
               } else {
-                return entity.state !== "off" ? "12px" : "0px";
+                if (entity.state == "off"){
+                  return "0px";
+                } else if (entity.state == "standby"){
+                  return "0px";
+                } 
+                else {
+                  return "12px";
+                }
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:59]: Does this also work?
```suggestion
                return (entity.state === "off" || entity.state === "standby") ? "12px" : "0px";
```

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 5]
```diff
@@ -68,7 +76,14 @@ card_media_player:
...
                 if(variables.ulm_card_media_player_collapsible){
-                  return entity.state === "off" ? "none" : "block";
+                  if (entity.state == "off"){
+                    return "none";
+                  } else if (entity.state == "standby"){
+                    return "none";
+                  } 
+                  else {
+                    return "block";
+                  }
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:86]:
```suggestion
                  return (entity.state === "off" || entity.state === "standby") ? "none" : "block";
```

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 6]
```diff
@@ -79,7 +94,14 @@ card_media_player:
...
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:104]:
```suggestion
                  return (entity.state === "off" || entity.state === "standby") ? "none" : "block";
```

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml (hunk 4 above)
Comment (mdb17) [reply custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:59]: I did not work for me. It always applies the 12px padding

Comment (mdb17) [reply custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:59]: This fixes it
`return (entity.state === "off" || entity.state === "standby") ? "0px" : "12px";`

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 7]
```diff
@@ -68,22 +68,18 @@ card_media_player:
             [[[
               if(variables.ulm_card_media_player_enable_controls) {
                 if(variables.ulm_card_media_player_collapsible){
-                  return entity.state === "off" ? "none" : "block";
+                return (entity.state === "off" || entity.state === "standby") ? "none" : "block";
                 }
-                return "block";
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:73]: This line is still needed

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 8]
```diff
@@ -68,22 +68,18 @@ card_media_player:
...
               }
-              return "none";
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:75]: This line is still needed

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 9]
```diff
@@ -68,22 +68,18 @@ card_media_player:
...
             ]]]
       item3:
         - display: |
             [[[
               if(variables.ulm_card_media_player_enable_volume_slider) {
                 if(variables.ulm_card_media_player_collapsible){
-                  return entity.state === "off" ? "none" : "block";
+                return (entity.state === "off" || entity.state === "standby") ? "none" : "block";
                 }
-                return "block";
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:84]: This line is still needed

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 10]
```diff
@@ -68,22 +68,18 @@ card_media_player:
...
               }
-              return "none";
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:86]: This line is still needed

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml (hunk 2 above)
Comment (basbruss) [reply custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:102]: See pr #878

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml (hunk 3 above)
Comment (basbruss) [reply custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:153]: See PR #878

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 11]
```diff
@@ -173,7 +161,14 @@ card_media_player:
...
                 - operator: "template"
-                  value: "[[[ return entity.state != 'off' ]]]"
+                  value: "[[[ return entity.state == 'standby' ]]]"
+                  name: |
+                    [[[
+                      let name = variables.ulm_card_media_player_name || states[entity.entity_id].attributes.friendly_name;
+                      return name;
+                    ]]]
+                - operator: "template"
+                  value: "[[[ return entity.state != 'off'  || 'standby']]]"
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:171]:
```suggestion
                  value: "[[[ return entity.state != 'off'  || entity.state != 'standby']]]"
```

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml (hunk 2 above)
Comment (mdb17) [reply custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:106]: @matrixx567 can you point me in the right direction on how to resolve this?

**Review by basbruss — APPROVED** [review]

**Pull request merged**